### DISCUSSION TITLE
vouchers between different events

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3344] [Q&A]
Hello pretix team!

Can I apply the same voucher in products of different events ?
Let's say I have eventA and eventB.
Also I have created productA.A and product A.B (of eventA) and productB.A and productB.B (of event B).
Is it possible to create vouchers that can apply to productA.A and productB.A ?
Or at least to all the products of both events ?

Thank you in advance

**Comment by raphaelm** [comment]
No, it is not possible
